package.json

```
@@ -2,7 +2,7 @@
   "name": "@tracktor/design-system",
   "description": "Tracktor Design System",
   "sideEffects": false,
-  "version": "4.17.0",
+  "version": "4.18.0",
```

Idem, un version de fix plutôt 
```suggestion
  "version": "4.17.1",
```